Runtimes/Resync.cmake

```diff
@@ -78,15 +78,11 @@ set(CoreLibs
   CompatibilityOverride
   stubs
   CommandLineSupport
-  core)
+  core
+  SwiftOnoneSUpport)
```

Comment:
```suggestion
  SwiftOnoneSupport)
```

Reply:
Thanks for catching this! I would have regressed builds on case sensitive file systems.